package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leaflet-distortableimage",
-  "version": "0.17.7",
```

Comment:
@jywarren I think this could have been a part of the problem, LDIs latest released version is `"0.16.6"` not `"0.17.7"`

Reply:
nevermind - realized 0.17.7 is up to date i just never published to npm